### PR TITLE
Allow multiple resource versions

### PR DESCRIPTION
**Pull request by da1nerd**
#### Describe what your pull request addresses (Please include relevant issue numbers):
related to https://github.com/unfoldingword/translationcore/issues/7121

Fixed a couple of bugs when comparing resource versions

#### Please include detailed Test instructions for your pull request:
-

#### Standard Test Instructions for PR Review Process:

- [ ] Double check unit tests that have been written
- [ ] Check for documentation for code changes
- [ ] Check that there are not inadvertent commits to tC Apps when reviewing a tC Core PR
- [ ] Checkout the branch locally and ensure that app runs as expected
  - [ ] Ensure tests pass
  - [ ] Open and watch the console for errors
  - [ ] Make sure all actions perform as expected
  - [ ] Import and Load a new Project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Switch project to an existing project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Next time reverse the order of importing after loading an existing project
- [ ] Reviewer should double check the DoD in the ISSUE, including the “spirit” of the story
- [ ] Ask Ben or Koz if you have any concerns about implementation (especially UI related)

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword/translationcore/7155)

### CODE REVIEW

File: src/js/helpers/ResourcesHelpers.js [hunk 1]
```diff
@@ -1152,13 +1150,15 @@ export function getMissingResources() {
         } else if (!fs.existsSync(userResourcePath)) {// if resource isn't found in user resources folder.
           copyAndExtractResource(staticResourcePath, userResourcePath, languageId, resourceId, resourceType);
         } else { // compare resources manifest modified time
-          const userResourceVersionPath = ResourceAPI.getLatestVersion(userResourcePath);
           const staticResourceVersionPath = ResourceAPI.getLatestVersion(staticResourcePath);
+          const version = path.basename(staticResourceVersionPath);
+          const userResourceVersionPath = path.join(userResourcePath, version);
+          const userResourceExists = fs.existsSync(userResourceVersionPath);
```
Comment (da1nerd) [inline src/js/helpers/ResourcesHelpers.js:1156]: Here we ensure we are comparing the same version in static and user resources.

**Comment by da1nerd** [comment]
I think this also fixes https://github.com/unfoldingword/translationcore/issues/7140

**Review by mannycolon — COMMENTED** [review]
Looks good but would like @PhotoNomad0 to review the code since he's more familiar with it.

**Comment by PhotoNomad0** [comment]
@da1nerd - In testing it appears to leave 2 ugnt versions out of 4 on my PC!

**Review by PhotoNomad0 — APPROVED** [review]
> Reviewed 1 of 1 files at r1, 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/unfoldingword/translationcore/7155)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @da1nerd)

**Pull request merged**